packages/titanium-dialog/src/titanium-dialog.ts

```diff
@@ -107,6 +108,7 @@ export class TitaniumDialogElement extends LitElement {
     click-trap {
       display: block;
       position: fixed;
+      z-index: 1;
```

Comment:
Without the transform styles the stacking context got messed up.  Adding z-index to insure the right layering.